src/components/generic/LogoBtn.tsx

```diff
@@ -0,0 +1,27 @@
+interface BtnProps {
+    name: string;
+    onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const LogoBtn = ({ name, onClick }: BtnProps) => {
+    return (
+        <div>
+            <button onClick={onClick} className="btn">
+                {name === 'play' ? (
+                    <img src={'/assignment-1-cohengmc/images/play-svgrepo-com.svg'} alt="play logo" className="btnLogo" />
```

Comment:
These should be `/public/images/...`

Reply:
@msaminsky I agree however I was having trouble with this when trying to get the images to display with GH Pages. When referencing the public folder as you suggest, the images show up on localhost but not when deployed on GH pages. Do you have any suggestions?